test/test_operations.py

```diff
@@ -454,6 +454,53 @@ def test_cat_empty_tensor(self):
     x_cat = torch.cat([x, empty_tensor_xla], 0)
     self.assertEqual(t_cat.data, x_cat.data.cpu())
 
+  def test_nan_to_num_in_place(self):
+    t = torch.tensor([float('nan'), float('nan'), -float('nan'), 3.14])
+
+    def fn(x):
+      x.nan_to_num_(1.0, 2.0, 3.0)
```

Comment:
I guess you need to return the result for comparison.

Reply:
I guess I really wasn't thinking when I wrote this haha 😅 Thanks!